DOL.WHD.Section14c.Web/src/modules/components/sectionWorkSites/tableConfig.js

```diff
@@ -5,17 +5,22 @@ var employeeColumns = [
       "data":null,
       "defaultContent": ""
   },
-  { title: 'Name', model: 'name' },
-  { title: 'Type of work performed', model: 'workType'  },
-  { title: 'Primary disability', model: 'primaryDisabilityId'  },
-  { title: 'How many jobs did this worker perform at this work site?', model: 'numJobs'  },
-  { title: 'Average # of hours worked per week on all jobs at this work site', model: 'avgWeeklyHours'  },
-  { title: 'Average earnings per hour for all jobs at this work site', model: 'avgHourlyEarnings'  },
-  { title: 'Prevailing wage rate for job described above', model: 'prevailingWage'  },
-  { title: 'Productivity measure/rating for job described above', model: 'hasProductivityMeasure'  },
-  { title: 'Commensurate wage rate/average earnings per hour for job described above', model: "commensurateWageRate" },
-  { title: 'Total hours worked for job described above', model: 'totalHours'  },
-  { title: 'Does worker perform work for this employer at any other work site?', model: 'workAtOtherSite'  },
+  { title: 'Name', data: 'name' },
+  { title: 'Type of work performed', data: 'workType'  },
+  { title: 'Primary disability', data: 'primaryDisabilityId'},
+  { title: 'How many jobs did this worker perform at this work site?', data: 'numJobs'  },
+  { title: 'Average # of hours worked per week on all jobs at this work site', data: 'avgWeeklyHours'  },
+  { title: 'Average earnings per hour for all jobs at this work site', data: 'avgHourlyEarnings'  },
+  { title: 'Prevailing wage rate for job described above', data: 'prevailingWage'  },
+  { title: 'Productivity measure/rating for job described above', data: 'hasProductivityMeasure', render: function(data){
+      if(data === null) {
+        return "n/a - piece rate"
+      }
+    }
+  },
+  { title: 'Commensurate wage rate/average earnings per hour for job described above', data: "commensurateWageRate" },
```

Comment:
The `data` property here is in double-quotes instead of single, but it looks like it's always been that way.  😛